app/src/main/java/com/hedvig/app/feature/marketing/data/MarketingRepository.kt

```diff
@@ -14,4 +16,8 @@ class MarketingRepository(
         .apolloClient
         .query(MarketingBackgroundQuery(defaultLocale(context).rawValue))
         .await()
+
+    fun hasSelectedMarket() = PreferenceManager
```

Comment:
This repository seems to have two widely different responsibilities - networking for fetching the background image and encapsulating the market preference on disk. Could it be appropriate to separate the responsibilities in two classes? 🤔